XCFramework/MUXSDKKaltura.xcframework/ios-arm64/MUXSDKKaltura.framework/Headers/MUXSDKKaltura-Swift.h

```diff
@@ -1,4 +1,4 @@
-// Generated by Apple Swift version 5.4.2 (swiftlang-1205.0.28.2 clang-1205.0.19.57)
+// Generated by Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
```

Comment:
interesting, why does this change? Is this bc our build machine has an earlier version of xcode than yours?

Reply:
yeah I think so.